pkg/sink/sink_test.go

```diff
@@ -257,6 +257,127 @@ func TestHandleEvent(t *testing.T) {
 	}
 }
 
+func TestHandleEventTriggerRef(t *testing.T) {
+	eventBody := json.RawMessage(`{"head_commit": {"id": "testrevision"}, "repository": {"url": "testurl"}, "foo": "bar\t\r\nbaz昨"}`)
+	numTriggers := 10
```

Comment:
Is there a reason to use 10 triggers to test this? It seems like the test will be simpler if we just use one trigger?

Reply:
Done.